code/Controllers/HomeController.cs

```diff
@@ -25,11 +25,12 @@ public HomeController(BagsApi bagsApi)
 		[Route("app/{*path}")]
 		public async Task<IActionResult> App(String path = null)
 		{
-			ViewData["url"] = "https://bagcupid.com/";
-			ViewData["type"] = "website";
-			ViewData["title"] = "Bag Cupid";
-			ViewData["description"] = "What is your dream bag? Are you having trouble finding it? Let us help you!";
-			ViewData["image"] = "https://bagcupid.com/img/logo/bagcupid.png";
+			var model = new MetaViewModel();
+			model.Url = "https://bagcupid.com/";
+			model.Type = "website";
+			model.Title = "Bag Cupid";
+			model.Description = "What is your dream bag? Are you having trouble finding it? Let us help you!";
+			model.Image = "https://bagcupid.com/img/logo/bagcupid.png";
```

Comment:
Style: We should move all of these defaults into the view model initialization since every instance of the view model should always have them set to something, no matter where it is instantiated.
```
public class IndexViewModel
{
	public String Url { get; set; } = "https://bagcupid.com/";
	public String Type { get; set; } = "website";
	public String Title { get; set; } = "Bag Cupid";
	public String Description { get; set; } = "What is your dream bag? Are you having trouble finding it? Let us help you!";
	public String Image { get; set; } = "https://bagcupid.com/img/logo/bagcupid.png";
}
```

This will make it so future development can't accidentally forget to set one of these, and also if we end up having multiple controllers/actions that return the index view, we don't have to worry about ensuring they all share the same default values.